modules/adglareBidAdapter.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+import {registerBidder} from '../src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'adglare';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['adglare'],
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.domain && bid.params.zID && bid.params.type);
+  },
+  buildRequests: function (validBidRequests, bidderRequest) {
+    let i;
+    let j;
+    let bidRequest;
+    let zID;
+    let domain;
+    let keywords;
+    let url;
+    let type;
+    let availscreen = window.innerWidth + 'x' + window.innerHeight;
+    let screen = ((window.devicePixelRatio || 1) * window.screen.width) + 'x' + ((window.devicePixelRatio || 1) * window.screen.height);
+    let sizes = [];
+    let serverRequests = [];
+    let timeout = bidderRequest.timeout || 0;
+    let referer = bidderRequest.refererInfo.referer || '';
+    let reachedtop = bidderRequest.refererInfo.reachedTop || '';
+    for (i = 0; i < validBidRequests.length; i++) {
+      bidRequest = validBidRequests[i];
+      zID = bidRequest.params.zID;
+      domain = bidRequest.params.domain;
+      keywords = bidRequest.params.keywords || '';
+      type = bidRequest.params.type;
+
+      // Build ad unit sizes
+      if (bidRequest.mediaTypes && bidRequest.mediaTypes[type]) {
+        for (j in bidRequest.mediaTypes[type].sizes) {
+          sizes.push(bidRequest.mediaTypes[type].sizes[j].join('x'));
+        }
+      } else {
```

Comment:
`bidRequest.sizes` is not supported in Prebid 3.

This is not needed in this PR into `master`. Will only be needed if you plan to this adapter to `2.44.x-legacy` branch

Reply:
This part of the code has been removed as requested.